site/content/_index.md

```diff
@@ -0,0 +1,72 @@
++++
+title = "wazero: the zero dependency WebAssembly runtime for Go developers"
+layout = "single"
++++
+
+WebAssembly is a way to safely run code compiled in other languages. Runtimes
+execute WebAssembly Modules (Wasm), which are most often binaries with a
+`.wasm` extension.
+
+wazero is the only zero dependency WebAssembly runtime written in Go. This
+means you can run code written in other languages that compile to WebAssembly
+modules.
+
+## Example
+
+The best way to learn wazero is by trying one of our [examples][1]
+
+For the impatient, here's how invoking a factorial function looks in wazero:
+
+```golang
```

Comment:
syntax highlighting seems not working correctly on the site :(

Reply:
fixed.. it likes "go" not "golang"